src/group/service.ts

```diff
@@ -30,3 +32,22 @@ export async function findAllGroupMembers(user: User): Promise<User[]> {
   return models;
 
 }
+
+export async function findByUserId(userId: number): Promise<any> {
+
+  const query = 'SELECT * FROM group_members WHERE user_id = ?';
+  const result = await database.query(query, [userId]);
+
+  if (result[0].length !== 1) {
+    throw new NotFound(`Can't find group member with User ID ${userId}`);
+  }
+
+}
+
+export async function save (userId: string, group: User): Promise<void> {
```

Comment:
I would expect a `save` function in a group save a group. This does something very different.